### ISSUE TITLE
Repeat first character on completion

### ISSUE THREAD

**Issue by Abukamel** [fallenhitokiri/autocomplete-plus-jedi#7]
Based on @joefitzgerald [comment](https://github.com/atom-community/autocomplete-plus/issues/307#issuecomment-79473404) on i open this issue for you to look into it.

**Comment by fallenhitokiri** [comment]
Thanks, I'll look into it.

**Comment by Abukamel** [comment]
Good Luck

**Comment by rvm521443** [comment]
You're doing some really strange stuff with the prefix. You should really use regex for that.
Something like:

``` coffeescript
regex = /[a-z|A-Z|_0-9-]+$/
prefix = options.prefix.match(regex)?[0] or ''
```

That's just for the top of my head though, but I'll submit a patch later if you want.

**Comment by fallenhitokiri** [comment]
the current prefix code makes sense considering some other planned features - but as a quick fix I agree that a simple regex would be fine. I won't get to this bug before mid next week so if you could submit a PR that would be great.

**Comment by rvm521443** [comment]
Check your email.

**Comment by fallenhitokiri** [comment]
Thank you! I'll check the patch and release a new version later today if it's working as expected.

**Comment by kosmodromov** [comment]
It isn't working as expected? ;) I have same problem.

**Comment by fallenhitokiri** [comment]
should be fixed with the 0.0.9 release thanks to @rvm521443

**Comment by fallenhitokiri** [comment]
as discussed here we try to consolidate all jedi providers - therefore this repo will be closed. https://github.com/tinloaf/autocomplete-plus-python-jedi/issues/16
